doc/changelogs/CHANGELOG_V13.md

```diff
@@ -42,6 +42,76 @@
   * [io.js](CHANGELOG_IOJS.md)
   * [Archive](CHANGELOG_ARCHIVE.md)
 
+<a id="13.12.0"></a>
+## 2020-03-24, Version 13.12.0 (Current), @mylesborins
+
+### Notable Changes
+
+* [[`a44da56254`](https://github.com/nodejs/node/commit/a44da56254)] - **(SEMVER-MINOR)** **doc**: update stability of report features (Colin Ihrig) [#32242](https://github.com/nodejs/node/pull/32242)
+* [[`306ed96c30`](https://github.com/nodejs/node/commit/306ed96c30)] - **(SEMVER-MINOR)** **doc,lib,src,test**: make --experimental-report a nop (Colin Ihrig) [#32242](https://github.com/nodejs/node/pull/32242)
+* [[`ea7f89da96`](https://github.com/nodejs/node/commit/ea7f89da96)] - **(SEMVER-MINOR)** **test**: remove common.skipIfReportDisabled() (Colin Ihrig) [#32242](https://github.com/nodejs/node/pull/32242)
+* [[`3f1f5189e5`](https://github.com/nodejs/node/commit/3f1f5189e5)] - **(SEMVER-MINOR)** **build**: make --without-report a no-op (Colin Ihrig) [#32242](https://github.com/nodejs/node/pull/32242)
+* [[`36ab39fc82`](https://github.com/nodejs/node/commit/36ab39fc82)] - **(SEMVER-MINOR)** **build**: remove node\_report option in node.gyp (Colin Ihrig) [#32242](https://github.com/nodejs/node/pull/32242)
+* [[`514b7c2744`](https://github.com/nodejs/node/commit/514b7c2744)] - **(SEMVER-MINOR)** **src**: unconditionally include report feature (Colin Ihrig) [#32242](https://github.com/nodejs/node/pull/32242)
+* [[`435fbbccb4`](https://github.com/nodejs/node/commit/435fbbccb4)] - **(SEMVER-MINOR)** **worker**: allow URL in Worker constructor (Antoine du HAMEL) [#31664](https://github.com/nodejs/node/pull/31664)
+* [[`975d6b0144`](https://github.com/nodejs/node/commit/975d6b0144)] - **(SEMVER-MINOR)** **util**: use a global symbol for `util.promisify.custom` (ExE Boss) [#31672](https://github.com/nodejs/node/pull/31672)
+
+### Commits
+
+* [[`3f1f5189e5`](https://github.com/nodejs/node/commit/3f1f5189e5)] - **(SEMVER-MINOR)** **build**: make --without-report a no-op (Colin Ihrig) [#32242](https://github.com/nodejs/node/pull/32242)
+* [[`36ab39fc82`](https://github.com/nodejs/node/commit/36ab39fc82)] - **(SEMVER-MINOR)** **build**: remove node\_report option in node.gyp (Colin Ihrig) [#32242](https://github.com/nodejs/node/pull/32242)
+* [[`2428afb9f4`](https://github.com/nodejs/node/commit/2428afb9f4)] - **build**: disable libstdc++ debug containers globally (Ben Noordhuis) [#30147](https://github.com/nodejs/node/pull/30147)
+* [[`dc5d4457bb`](https://github.com/nodejs/node/commit/dc5d4457bb)] - **build**: macOS package notarization (Rod Vagg) [#31459](https://github.com/nodejs/node/pull/31459)
+* [[`5366682875`](https://github.com/nodejs/node/commit/5366682875)] - **deps**: update npm to 6.14.3 (Myles Borins) [#32368](https://github.com/nodejs/node/pull/32368)
```

Comment:
We should likely point out in the notable changes that this update fixes potential warnings about a security issue in the npm dependencies. Including a note that this security issue does not apply to npm directly.

Reply:
I think we should skip pointing this out more explicitly. we had no security issues in node core or in npm. The warning was annoying yes, but do we need to bring more attention to ti?